packages/remix-react/components.tsx

```diff
@@ -483,10 +485,10 @@ function PrefetchPageLinksImpl({
       {moduleHrefs.map((href) => (
         <link key={href} rel="modulepreload" href={href} {...linkProps} />
       ))}
-      {styleLinks.map((link) => (
+      {keyedPrefetchLinks.map(({ key, link }) => (
         // these don't spread `linkProps` because they are full link descriptors
         // already with their own props
-        <link key={link.href} {...link} />
+        <link key={key} {...link} />
```

Comment:
This fixes an unreported bug where responsive image prefetch links would all have a key of `undefined` since they have `imgSrcSet` and `imgSizes` properties but no `href` property.